RELEASES.md

```diff
@@ -232,6 +240,21 @@ completed, open a PR using the process above.
 Only when the bug is not seen in main and must be made for the specific
 release branch should you open a PR with new code.
 
+### Upgrade Path
+
+The upgrade path for containerd is such that the 0.0.x patch releases are
+always backward compatible with its major and minor version. Minor (0.x.0)
+version will always be compatible with the previous minor release. i.e. 1.2.0
+is backwards compatible with 1.1.0 and 1.1.0 is compatible with 1.0.0. There is
+no compatibility guarantees for upgrades that span multiple, _minor_ releases.
+For example, 1.0.0 to 1.2.0 is not supported. One should first upgrade to 1.1,
+then 1.2.
+
+There are no compatibility guarantees with upgrades to _major_ versions. For
```

Comment:
This should probably be updated to discuss the upgrade path from 1.7.x to 2.0.x.

We don't need to make it a part of this change, but we also need to cover LTS upgrades; i.e., can you upgrade directly from one LTS to the next?

Reply:
Yeah, we should revisit this section